test/trento/application/integration/discovery/policies/sap_system_policy_test.exs

```diff
@@ -86,6 +89,58 @@ defmodule Trento.Integration.Discovery.SapSystemPolicyTest do
             ]} =
              "sap_system_discovery_application_diagnostics"
              |> load_discovery_event_fixture()
-             |> SapSystemPolicy.handle()
+             |> SapSystemPolicy.handle([])
+  end
+
+  describe "delta deregistration" do
```

Comment:
Other possible test would be to check that no commands are sent if we don't have currently instances and the coming payload is empty of sap instances as well.

Reply:
indeed, it doesn't need to be part of the delta group of tests.
Yes, it would look like the same than in the cluster